docs/source/cql.rst

```diff
@@ -48,3 +56,41 @@ A ``between`` example for a specific property through an HTTP POST request:
       "upper": 0.60
     }
   }'
+
+Or 
+
+.. code-block:: bash
+
+  curl --location --request POST 'https://ogcapi.bgs.ac.uk/collections/recentearthquakes/items?f=json&limit=10&filter-lang=cql-json' 
```

Comment:
Update URLs to be more generic (i.e. localhost)

Reply:
Fixed in latest commits by @ximenesuk and @KoalaGeo.